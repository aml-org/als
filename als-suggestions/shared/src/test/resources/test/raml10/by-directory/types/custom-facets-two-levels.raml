#%RAML 1.0
title: test
types:
  CustomDate:
    facets:
      onlyFutureDates?: boolean
      noHolidays: boolean
  AnotherDate:
    type: CustomDate
    facets:
      aNewFacet: boolean
  PossibleMeetingDate:
    type: AnotherDate
    *