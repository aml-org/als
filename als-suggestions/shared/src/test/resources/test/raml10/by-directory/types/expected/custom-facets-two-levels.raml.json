[
  {
    "label": "aNewFacet",
    "kind": [],
    "detail": "unknown",
    "documentation": "aNewFacet",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "aNewFacet: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "default",
    "kind": [],
    "detail": "parameters",
    "documentation": "default",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "default: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "description",
    "kind": [],
    "detail": "docs",
    "documentation": "description",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "description: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "displayName",
    "kind": [],
    "detail": "docs",
    "documentation": "displayName",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "displayName: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "enum",
    "kind": [],
    "detail": "parameters",
    "documentation": "enum",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "enum:\n      - "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "example",
    "kind": [],
    "detail": "docs",
    "documentation": "example",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "example:\n      "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "examples",
    "kind": [],
    "detail": "docs",
    "documentation": "examples",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "examples:\n      "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "facets",
    "kind": [],
    "detail": "unknown",
    "documentation": "facets",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "facets:\n      "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "maxLength",
    "kind": [],
    "detail": "parameters",
    "documentation": "maxLength",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "maxLength: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "minLength",
    "kind": [],
    "detail": "parameters",
    "documentation": "minLength",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "minLength: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "noHolidays",
    "kind": [],
    "detail": "unknown",
    "documentation": "noHolidays",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "noHolidays: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "onlyFutureDates",
    "kind": [],
    "detail": "unknown",
    "documentation": "onlyFutureDates",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "onlyFutureDates: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "pattern",
    "kind": [],
    "detail": "parameters",
    "documentation": "pattern",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "pattern: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "xml",
    "kind": [],
    "detail": "schemas",
    "documentation": "xml",
    "deprecated": [],
    "preselect": [],
    "sortText": null,
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      2
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 13,
          "character": 4
        },
        "end": {
          "line": 13,
          "character": 4
        }
      },
      "newText": "xml:\n      "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]